android/pytorch-native/README.md

```diff
@@ -0,0 +1,137 @@
+
+# Pytorch Android Build Procedure
+## Linux (Ubuntu 20.04) android-sdk install
+```
```

Comment:
In markdown, adjacent lines are considered part of the same paragraph. So, this says that the top level heading, second level heading, and code block are in the same paragraph. Instead, what you want is an empty line separating these different pieces.

Now, most parsers will actually interpret this correctly anyway. However, the one we use to build our docs site (https://docs.djl.ai) doesn't. So you will actually need to add all the separating line breaks here to have it render properly